### ISSUE TITLE
[bug-website module] HeavenManga website problem

### ISSUE THREAD

**Issue by AdrianChandra** [dazedcat19/FMD2#97]
Download succesful but the content is koomanga.com image placeholder and also the url is now ww3.heaventoon.com

**Comment by Slasar41** [comment]
Which title does that?
I tried with Solo Leveling latest chapter and the content is fine.

**Comment by AdrianChandra** [comment]
> 
> 
> Which title does that?
> I tried with Solo Leveling latest chapter and the content is fine.

I tried solo leveling chap 117 only got 111kb of cbz file and all the content is koomanga.com png. All manga downloaded is like that.

![content](https://user-images.githubusercontent.com/38415103/92321654-c7037380-f055-11ea-945c-e008775b0f31.png)

Tried updating heavenmanga.lua to ww3.heaventoon.com download keep downloading.... never ending.
Tried updating heavenmanga.lua to ww4.koomanga.com everything is OK, all manga downloaded ok now, is it the right solution?
